airbyte-cdk/python/airbyte_cdk/sources/declarative/declarative_component_schema.yaml

```diff
@@ -1773,9 +1773,22 @@ definitions:
         title: Record Filter
         description: Responsible for filtering records to be emitted by the Source.
         "$ref": "#/definitions/RecordFilter"
+      schema_normalization:
+        "$ref": "#/definitions/SchemaNormalization"
       $parameters:
         type: object
         additionalProperties: true
+  SchemaNormalization:
+    title: Schema Normalization
+    description: Responsible for normalization according to the schema.
+    type: string
+    enum:
+      - no
```

Comment:
let's call this NO_TRANSFORM

Reply:
I assume this is the option that user will see in the UI, right? wouldn't "No Trasform" will look better in that case?

Reply:
good question. @lmossman does the FE rename enums or show them as they are in the schema?

Reply:
The Connector Builder currently just shows enum values as they are, e.g. for HTTP Method we show `GET` and `POST` in the UI. 

But it also would be pretty low effort to rename the values when adding them to the builder UI

Reply:
thanks Lake! @keu let's go with something user-friendly then.

Thinking more about the name, I sort of dislike normalization because the name clashes with another Airbyte concept.

How about 
- `None`
- `Default`